src/fev/task.py

```diff
@@ -376,15 +377,20 @@ def __post_init__(self):
         assert all(0 < q < 1 for q in self.quantile_levels), "All quantile_levels must satisfy 0 < q < 1"
         self.quantile_levels = sorted(self.quantile_levels)
 
-        self.eval_metric = self.eval_metric.upper()
-        self.extra_metrics = [m.upper() for m in self.extra_metrics]
-        for metric in [self.eval_metric] + self.extra_metrics:
-            if metric not in AVAILABLE_METRICS:
-                raise ValueError(
-                    f"Evaluation metric '{metric}' is not available. Available metrics: {sorted(AVAILABLE_METRICS)}"
-                )
-            if metric in QUANTILE_METRICS and len(self.quantile_levels) == 0:
-                raise ValueError(f"Please provide quantile_levels when using a quantile metric '{metric}'")
+        metrics = [get_metric(m) for m in [self.eval_metric] + self.extra_metrics]
+
+        metric_names = [m.name for m in metrics]
```

Comment:
Should we also maybe expose the name of the metrics as an attribute of `Task` of type `list[str]`? This would be helpful for knowing which extra will be available in the evaluation summary without needing import `get_metric`.